tests/trunc-double-uniform.ispc

```diff
@@ -0,0 +1,127 @@
+
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+
+    uniform float error = 0;
+    RET[programIndex] = 0;
+    uniform double deltaLow = 0.000001d;
+    uniform double deltaHigh = 0.999999d;
```

Comment:
No need to make these values small / close to 1. `0.2` and `0.8` would be better.
Small / close to 1 values are confusing and might make people think that this is somehow important and means something.

Reply:
Done

Reply:
Two tests still not updated.